docs/guides/nextjs-page-router-feature-flags.mdx

```diff
@@ -0,0 +1,200 @@
+---
+sidebar_label: How to set up Feature Flags with Next.js (Page Router)
+title: How to set up Feature Flags with Next.js (Page Router)
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Next.js is an open-source React framework for building fast and user-friendly full-stack web applications.
+
+In this tutorial we'll integrate our [React Client SDK](https://docs.statsig.com/client/reactSDK) and [NodeJS Server SDK](https://docs.statsig.com/server/nodejsServerSDK) into a simple Next.js app with Page Router and check a feature gate to determine if the end user should see a new feature. 
+
+## Create a Feature Gate in Statsig
+
+Create a Statsig account and obtain an SDK key: [Signup for free](https://www.statsig.com/signup)!
+
+The below command can be run to automatically create the simple `nextjs_app_bg` gate in Statsig that gets used throughout this integration guide and demo app. The gate randomizes based on `userID` and has a 50/50 allocation. Make sure to replace the authorization key in the header with your key generated in Statsig project settings as documented [here](https://docs.statsig.com/console-api/introduction).
+```bash
+curl --request POST 'https://statsigapi.net/console/v1/gates' \
```

Comment:
I know it's more work but imo we should walkthrough with screenshots of console instead of this power user API call, since we want to teach newcomers to statsig who find this page via google how easy gates are to use and what they look like

Reply:
fair, probably worth creating a generic copy for this step to be used across all frameworks.